src/toGamut.js

```diff
@@ -15,84 +17,91 @@ import getColor from "./getColor.js";
  * @param {Object|string} options object or spaceId string
  * @param {string} options.method - How to force into gamut.
  *        If "clip", coordinates are just clipped to their reference range.
+ *        If "css", coordinates are reduced according to the CSS 4 Gamut Mapping Algorithm.
  *        If in the form [colorSpaceId].[coordName], that coordinate is reduced
  *        until the color is in gamut. Please note that this may produce nonsensical
  *        results for certain coordinates (e.g. hue) or infinite loops if reducing the coordinate never brings the color in gamut.
  * @param {ColorSpace|string} options.space - The space whose gamut we want to map to
  */
-export default function toGamut (color, {method = defaults.gamut_mapping, space = color.space} = {}) {
+
+export default function toGamut (color, { method = defaults.gamut_mapping, space = color.space } = {}) {
 	if (util.isString(arguments[1])) {
 		space = arguments[1];
 	}
 
 	space = ColorSpace.get(space);
 
-	if (inGamut(color, space, {epsilon: 0})) {
-		return getColor(color);
-	}
-
 	// 3 spaces:
 	// color.space: current color space
 	// space: space whose gamut we are mapping to
 	// mapSpace: space with the coord we're reducing
+
 	let spaceColor = to(color, space);
+	if (method === "css") {
+		spaceColor = to(toGamutCSS(color, space), color.space);
```

Comment:
@jamesnw Should this be:

```suggestion
		spaceColor = to(toGamutCSS(color, {space}), color.space);
```